packages/validator/src/validator.constants.js

```diff
@@ -0,0 +1,52 @@
+//@flow
+
+export const FIELD_TYPE = {
+  NUMBER: 'NUMBER',
+  TEXT: 'TEXT',
+  DATE: 'DATE',
+  SWITCH: 'SWITCH',
+  FILE: 'FILE',
+  RELATION: 'RELATION',
+};
+
+export const TEXT_FORMAT = {
+  UNFORMATTED: 'UNFORMATTED',
+  NAME: 'NAME',
+  ADDRESS: 'ADDRESS',
+  SSN: 'SSN',
+  EIN: 'EIN',
+  PHONE: 'PHONE',
+  EMAIL: 'EMAIL',
+};
+
+export const TEXT_FORMAT_PATTERN = {
+  [TEXT_FORMAT.EMAIL]: /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i,
+  [TEXT_FORMAT.PHONE]: /^(0|[1-9][0-9]{9})$/i,
+  [TEXT_FORMAT.SSN]: /^(?!666|000|9\d{2})\d{3}[- ]{0,1}(?!00)\d{2}[- ]{0,1}(?!0{4})\d{4}$/,
+  [TEXT_FORMAT.EIN]: /^\d{2}[- ]{0,1}\d{7}$/,
+};
+
+export const DATE_FORMAT = {
+  DATE: 'DATE',
+  DATETIME: 'DATETIME',
+};
+
+export const DATE_FORMAT_PATTERN = {
+  [DATE_FORMAT.DATE]: /\d{4}-[01]\d-[0-3]\d/,
+  [DATE_FORMAT.DATETIME]: /\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z)/,
+};
+
+export const ERROR = {
```

Comment:
What are you think about extend info in validators texts?

For example:
As user, I want know min value if I see error "Value is lower than minimum allowed value".
As user, I want know max value if I see error "Value is greater than maximum allowed value".
etc...

Reply:
Sounds good. I'll implement this.